local/alpha-sdk.dockerfile

```
@@ -0,0 +1,14 @@
+ARG SDK
+ARG HOST_GOARCH
+
+FROM $SDK
+FROM --platform=linux/${HOST_GOARCH} $SDK
+
+COPY build/rpms/ /twoliter/alpha/build/rpms/
```

Should we limit this to RPMs for the target arch?

```suggestion
ARG HOST_GOARCH
ARG TARGET_ARCH

FROM $SDK
FROM --platform=linux/${HOST_GOARCH} $SDK

COPY build/rpms/*.${TARGET_ARCH}.rpm /twoliter/alpha/build/rpms/
```